main.py

```diff
@@ -272,9 +245,7 @@ def resource_gone(e: BaseException) -> Tuple[Response, int]:
         "Names.conf",
     ]
 
-    dirs: List[str] = list(
-        map(os.path.dirname, [__file__, reynir.__file__, reynir_correct.__file__])  # type: ignore
-    )
+    dirs: List[str] = list(map(os.path.dirname, [__file__, reynir.__file__]))  # type: ignore
```

Comment:
I think type:ignore might no longer be needed - it's probably there because of an early bug in Pylance.